server/routes/auth.js

```diff
@@ -21,6 +21,11 @@ router.get(
 		const user = await User.findById(req.user.id).select(
 			'-password'
 		);
+		if (!user) {
```

Comment:
Does the `findById` return `null` or the `select`?

If it's the find then we could get a null reference error here.